tasks/self-introduction/self-introduction.md

```diff
@@ -1,59 +1,59 @@
-4. В созданной ветке `self-introduction` создайте папку `self-introduction`, в ней создайте файл `index.md`.
-5. Добавьте ссылку на ваше YouTube-видео в файл `index.md`
-6. Под ссылкой добавьте текстовую расшифровку вашего выступления. Текст должен повторять сказанное в вашем выступлении, но может быть не дословным — можно отредактировать текст, убрав из него лишние междометия, слова-паразиты и т.п.
-7. Для деплоя используйте gh-pages [Как сделать деплой задания из приватного репозитория школы](https://docs.rs.school/#/private-repository?id=Как-сделать-деплой-задания-из-приватного-репозитория-школы)
-8. Сделайте Pull Request из ветки `self-introduction` в ветку `main`. Pull Request должен называться так же, как и задание — **Self-Introduction**, и соответствовать [требованиям к Pull Requests](https://docs.rs.school/#/pull-request-review-process?id=Требования-к-pull-request-pr). В качестве скриншота результата можно прикрепить скриншот видео, а в качестве ссылки на деплой — ссылку вида `https://rolling-scopes-school.github.io/your-github-account-JSFE2022Q3/self-introduction/`. **Мержить Pull Request не нужно**.
+1. Record your presentation in English. It is desirable that your face is in the frame (for recording, you can use a webcam, phone camera, video camera, laptop camera, etc.).
+2. Upload the video to your personal [YouTube](https://www.youtube.com/) account. You can make a video available only via a link - to do this, in the video settings, set [Параметры доступа > Доступ по ссылке (Visibility > Unlisted)](https://support.google.com/youtube/answer/157177?co=GENIE.Platform%3DDesktop&hl=en).
+3. In your **private** repository (using a link like `https://github.com/rolling-scopes-school/your-github-account-JSFE2022Q3`) create a `self-introduction` branch from the `main` branch.
+4. In the created branch `self-introduction` create a folder `self-introduction` with a file `index.md` inside.
+5. Add a link to your YouTube video to the `index.md` file
+6. Add a text transcript of your presentation below the link. The text should repeat what was said in your presentation, but may not be verbatim - you can edit the text by removing unnecessary interjections, parasitic words, etc. from it.
+7. Use gh-pages for deployment [Как сделать деплой задания из приватного репозитория школы](https://docs.rs.school/#/private-repository?id=Как-сделать-деплой-задания-из-приватного-репозитория-школы)
+8. Make a Pull Request from the `self-introduction` branch to the `main` branch. The Pull Request must be named the same as the task, **Self-Introduction**, and conform to the [Requirements for Pull Requests](https://docs.rs.school/#/pull-request-review-process?id=Требования-к-pull-request-pr). As a screenshot of the result, you can attach a screenshot of the video, and as a link to the deployment use a link like `https://rolling-scopes-school.github.io/your-github-account-JSFE2022Q3/self-introduction/`. **Don't merge the Pull Request**.
```

Comment:
Create a Pull Request
must be named -> should be named
https://docs.app.rs.school/#/platform/pull-request-review-process?id=pull-request-requirements-pr - from EN docs